model/resource/deployment_environment.yaml

```diff
@@ -1,22 +1,8 @@
 groups:
   - id: deployment
-    prefix: deployment
     type: resource
     brief: >
       The software deployment.
     attributes:
-      - id: environment
-        type: string
-        stability: experimental
-        brief: >
-          Name of the [deployment environment](https://wikipedia.org/wiki/Deployment_environment)
-          (aka deployment tier).
-        note: |
-          `deployment.environment` does not affect the uniqueness constraints defined through
-          the `service.namespace`, `service.name` and `service.instance.id` resource attributes.
-          This implies that resources carrying the following attribute combinations MUST be
-          considered to be identifying the same service:
-
-          * `service.name=frontend`, `deployment.environment=production`
-          * `service.name=frontend`, `deployment.environment=staging`.
-        examples: ['staging', 'production']
+      - ref: deployment.environment
+        requirement_level: recommended
```

Comment:
the requirement level wasn't there before?

Reply:
No, it was all the default (recommended) but I think I like having it there, even if it's the default.